hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/physical/calcite/index/IndexComponentCandidate.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.hazelcast.sql.impl.calcite.opt.physical.index;
+package com.hazelcast.jet.sql.impl.opt.physical.calcite.index;
```

Comment:
Why suddenly introducing calcite in the middle of the package name?

Reply:
These classes work with Calcite-specific entities during index resolving process. That's the reason why I separated these classes into the specific  sub-package.

Reply:
All the classes in `opt` package deal with `Calcite`. Even `JetIndexResolver` which works with both indexes and `Calcite` and yet it's not in _calcite_ package.

Reply:
Okay, will revert the change. Thanks.